### ISSUE TITLE
584111F7 - Minecraft: Xbox 360 Edition Beta

### ISSUE THREAD

**Issue by ProgrammingHider** [xenia-project/game-compatibility#1362]
[Marketplace](https://marketplace.xbox.com/en-US/Product/Minecraft-Xbox-360-Edition/66acd000-77fe-1000-9115-d802584111f7?cid=search)

Tested on https://github.com/xenia-project/xenia/commit/bc2eaf9b

# Issues:
Crashes after "Loading Spawn Area"

# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/3438342/xenia.zip)

# Screenshot(s):
![xenia_gJie0TV6mZ](https://user-images.githubusercontent.com/38049865/61996730-ec266400-b097-11e9-9a0f-b09fa711f165.png)

# Labels:
state-menus

**Comment by TGLTClutch** [comment]
have you tried the D3D12 version?

**Comment by ProgrammingHider** [comment]
Yes I Have Tried the D3D12, but that Crashes before even been able to Load a Game

And I'm on 8.1, just using Aero8 which is a modified Version with a lot of Aero

**Comment by illusion98** [comment]
D3D12 version of Xenia will only work on Windows 10.

**Comment by TGLTClutch** [comment]
> 
> 
> Yes I Have Tried the D3D12, but that Crashes before even been able to Load a Game
> 
> And I'm on 8.1, just using Aero8 which is a modified Version with a lot of Aero
> 
> edit 1: No It's the a Beta from 2012, got it from [REDACTED]

![image](https://user-images.githubusercontent.com/37595685/61997338-85f20f00-b0a0-11e9-8443-8a5f37f7eb93.png)
So this one, I mean how's a 70MB game even supposed to run xD

**Comment by ProgrammingHider** [comment]
Correct, it's that one

**Comment by ProgrammingHider** [comment]
Update:
The Game Also Crashes with D3D12 but i dont get a "Xenia.exe Stopped working" error

**Comment by brandonlee04** [comment]
ive got it to work you have to just press leaderboards then achievements and finally press play and it should run 
![xbox 360 minecraft beta](https://user-images.githubusercontent.com/12752498/62734717-90f25b00-ba21-11e9-8267-f99db162fb7b.PNG)

**Comment by Margen67** [comment]
@ProgrammingHider Was this ever released?

**Comment by ProgrammingHider** [comment]
Yes and No, Yes because they Eventually Turned into a Fully Developed Game. And No Because this is a leaked Beta, this beta version was never avalible to the Public

**Comment by ProgrammingHider** [comment]
@brandonlee04 thanks for figuring out how to play it. Also it Looks Similar to The Final Game, eventhough this is a early beta. Very Interesting

**Comment by dogyteryian** [comment]
Ok where do i need to download this?????

**Comment by Margen67** [comment]
@dogyteryian I'd tell you but you wouldn't read anyway: https://github.com/xenia-project/game-compatibility#what-not-to-post

**Comment by Pakiucs151** [comment]
Seems like its fully working now, i had no issues on gameplay
Tested on old hardware (i7 3770, GTX 680, 12GB Ram, 1tb WD Blue)


![Captura MCXDK 2](https://user-images.githubusercontent.com/66497859/83972135-2b226a00-a8a4-11ea-9f2d-3777094ca930.PNG)
![Captura MCXDK](https://user-images.githubusercontent.com/66497859/83972136-2bbb0080-a8a4-11ea-9a39-1744d6a83386.PNG)

**Comment by Trainboat555** [comment]
Does anyone have any info on where to get the beta as I'm having a large amount of trouble finding it

**Comment by Strider-Aragorn** [comment]
Good Day,

I'm trying to run Minecraft Xbox 360 complete edition (584111F7) along with TU, on Xenia Canary (as Master does not support TU); all seems good, have created a world, saved the played game but unfortunately, the saved game is not loading again and crashing the emulator with error (The guest has crashed)


![Minecraft-TU](https://user-images.githubusercontent.com/70264790/91292600-2f2d9c00-e7b0-11ea-97c8-985963228fb0.jpg)

![Minecraft-TU-Loading](https://user-images.githubusercontent.com/70264790/91292625-381e6d80-e7b0-11ea-8960-fa7f7d705fb0.jpg)

![Minecraft-TU-Loading-Crashed](https://user-images.githubusercontent.com/70264790/91292640-3ce32180-e7b0-11ea-8e7b-36872f244d12.jpg)

**Comment by Sterophonick** [comment]
I think this game broke at some point...
![Screenshot 2020-10-08 095140](https://user-images.githubusercontent.com/34801996/95489648-ff93b700-094b-11eb-857b-57c7974c724c.png)
![Screenshot 2020-10-08 095213](https://user-images.githubusercontent.com/34801996/95489653-028ea780-094c-11eb-922a-b81514dc4ae6.png)
![unknown](https://user-images.githubusercontent.com/34801996/95489671-07535b80-094c-11eb-8df1-7b4976d9bdb2.png)
![Screenshot 2020-10-08 095403](https://user-images.githubusercontent.com/34801996/95489877-484b7000-094c-11eb-8420-36cc7852b04a.png)

Tested on https://github.com/xenia-project/xenia/commit/2cebd3cabecfeedc643b4116debcb3fa6c5cc33e

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5349571/xenia.zip)

**Comment by YellowPigIL** [comment]
i have a question.




WHERE CAN I DOWNLOAD IT???

**Comment by akineechan** [comment]
> WHERE CAN I DOWNLOAD IT???

if you search around you may find a copy. i did share a link before however it was redacted

**Comment by Arysr** [comment]
There are 4 known existing builds of this beta, and 3 that are public, according to the [Minecraft Wiki](https://minecraft.fandom.com/wiki/Xbox_360_Edition_0.66.0054.0)

The bottom left of the "controls" menu has the version build number of the beta you have.

I have the same build as @Sterophonick 

I know this because the only known copy of this version is edited with those different textures you see in their screenshots.
This is build 1.66.0033.0.
It works fine for me.


![xenia (detached_10ff77a2_Jun  3 2021) 6_5_2021 12_49_53 AM](https://user-images.githubusercontent.com/70122603/120881710-cd992a00-c598-11eb-8412-a79f85e76129.png)
![xenia (detached_10ff77a2_Jun  3 2021) 6_5_2021 12_50_22 AM](https://user-images.githubusercontent.com/70122603/120881715-d427a180-c598-11eb-875b-049be6fac6a1.png)

The textures appearing as their normal counterparts at a distance or at sharp angles (seen in the 2nd screenshot is due to the game using lower resolution textures as an optimization feature. Only the

**Comment by ridge4mama** [comment]
> Does anyone have any info on where to get the beta as I'm having a large amount of trouble finding it

https://marketplace.xbox.com/

**Comment by Arysr** [comment]
> > Does anyone have any info on where to get the beta as I'm having a large amount of trouble finding it
> 
> https://marketplace.xbox.com/

Aint gonna be on there. Seems the only way to have ever obtained this version is through illegitimate means, whether you got it today or when it was leaked.

**Comment by loneneuron** [comment]
@Trainboat555 Not sure if it's the Beta release you guys talking about, but here's the 0.66.0062.0 build uploaded onto the Internet Archive only for archival purposes:
https://archive.org/details/minecraft-xbox-360-edition-base-game
You can find other beta builds there as well.

**Comment by adhe482** [comment]
Help how to fix this error

there's been an issue content from game disc. this is likely caused bad or unimplemented file IO calls
![minecraft disc error](https://user-images.githubusercontent.com/41714439/233556047-1f5c4fab-f26d-406f-845c-e7dbbb634582.png)
